### PR TITLE
Set accurate template for tenant root namespaces

### PR DESCRIPTION
**Pull request by yokaze**
This PR adds `init-template` Application that deploys `init-template` namespace.
It is annotated as a template namespace of Accurate, that stores common resources.

Signed-off-by: Daichi Sakaue <daichi-sakaue@cybozu.co.jp>

### CODE REVIEW

File: accurate/base/tenant-template.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```
Comment (ymmt2005) [inline accurate/base/tenant-template.yaml:1]: accurate アプリにふくまれるべきではないと思います。namespaces 辺りでは。

File: accurate/base/kustomization.yaml [hunk 2]
```diff
@@ -2,6 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - upstream/default
+- tenant-template.yaml
```
Comment (ymmt2005) [inline accurate/base/kustomization.yaml:5]: ditto

File: accurate/base/tenant-template.yaml (hunk 1 above)
Comment (yokaze) [reply accurate/base/tenant-template.yaml:1]: @ymmt2005 namespaces の下だと `team: neco` が付いてしまうみたいですね。。それを避けようとすると新しいアプリを作る必要があるかもです。。

**Review by ymmt2005 — APPROVED** [review]
Seems OK, but I have a question.

Will namespaces such as `app-kodama` become a sub-namespace of Accurate?

**Comment by yokaze** [comment]
@ymmt2005 I think such namespaces may be the root namespaces derived from `init-template`, but I will investigate the appropriate strategy for it.

**Pull request merged**